docs/api/ShallowWrapper/simulate.md

```diff
@@ -53,3 +53,8 @@ expect(wrapper.find('.clicks-1').length).to.equal(1);
 - Currently, event simulation for the shallow renderer does not propagate as one would normally
 expect in a real environment. As a result, one must call `.simulate()` on the actual node that has
 the event handler set.
+- Even though the name would imply this simulates an actual event, `.simulate()` will in fact 
+target the component's prop based on the event you give it. For example, `.simulate('click')` will
+actually get the `onClick` prop and call it. This is important to keep in mind when using
+libraries such as MaterialUI where clicks are set on a `onTouchTap` prop. You will need to call
```

Comment:
Maybe generalize this so it doesn't mention a specific library? Just for those who don't know what it is. If not, it should be Material-UI (with the dash), per their website.